crates/holochain/tests/tests/regression/mod.rs

```diff
@@ -114,39 +114,15 @@ async fn zome_with_no_link_types_does_not_prevent_delete_links() {
 }
 
 #[tokio::test(flavor = "multi_thread")]
-#[ignore = "flaky"]
 async fn zero_arc_can_link_to_uncached_base() {
-    use hdk::prelude::*;
-
     holochain_trace::test_run();
 
     let empty_arc_conductor_config = SweetConductorConfig::rendezvous(false)
         .no_dpki_mustfix()
         .tune_network_config(|nc| nc.target_arc_factor = 0);
 
-    let other_config = SweetConductorConfig::rendezvous(false)
```

Comment:
There was another conductor with Bob, a gossip partner for Alice so that she could declare a full arc organically. But that lead to a problem later on in the test, because Bob would declare a full storage arc too, but wouldn't hold Alice's ops after she created entries straight away, so the part where Carol links to Alice's actions was flaky.